packages/amplify-category-auth/tests/commands/update.test.js

```diff
@@ -0,0 +1,173 @@
+
+jest.mock('fs', () => ({
```

Comment:
That's actually what I tried initially, but it did not work.

Reply:
What error did you get?

Reply:
I believe it was json error ('unexpected character at line 1').

Reply:
Thats because if you don't define any mock return values, jest returns undefined. You could update the code to this and it should not throw any errors
```javascript
const { readFileSync } = require('fs')

jest.mock('fs')
readFileSync.mockReturnValue({})
```
Only reason I prefer to use `jest.mock` without creating an implementation is that it is easier to add/use additional method in the module easily